.github/workflows/dispatch-rebottle.yml

```diff
@@ -28,6 +28,7 @@ on:
 
 permissions:
   contents: read
+  pull-requests: write
```

Comment:
May want to add this to the `bottle` job's `permissions` instead. Some permissions are already set there.

https://github.com/Homebrew/homebrew-core/blob/c01caa6aed75be742b3a2b1ff2bb2bba19b90329/.github/workflows/dispatch-rebottle.yml#L57-L60

By the way, the `bottle` job in `dispatch-build-bottle` workflow may need this as well.